test/PowerShellEditorServices.Test/PsesHostFactory.cs

```diff
@@ -53,25 +45,24 @@ public static PsesInternalHost Create(ILoggerFactory loggerFactory)
             }
 
             HostStartupInfo testHostDetails = new(
-                "PowerShell Editor Services Test Host",
-                "Test.PowerShellEditorServices",
-                new Version("1.0.0"),
+                name: "PowerShell Editor Services Test Host",
+                profileId: "Test.PowerShellEditorServices",
+                version: new Version("1.0.0"),
                 psHost: new NullPSHost(),
-                TestProfilePaths,
+                profilePaths: TestProfilePaths,
                 featureFlags: Array.Empty<string>(),
                 additionalModules: Array.Empty<string>(),
-                initialSessionState,
+                initialSessionState: initialSessionState,
                 logPath: null,
-                (int)LogLevel.None,
+                logLevel: (int)LogLevel.None,
                 consoleReplEnabled: false,
                 usesLegacyReadLine: false,
                 bundledModulePath: BundledModulePath);
 
             var psesHost = new PsesInternalHost(loggerFactory, null, testHostDetails);
 
             // NOTE: Because this is used by constructors it can't use await.
-            // TODO: Should we actually load profiles here?
-            if (psesHost.TryStartAsync(new HostStartOptions { LoadProfiles = true }, CancellationToken.None).GetAwaiter().GetResult())
+            if (psesHost.TryStartAsync(new HostStartOptions { LoadProfiles = false }, CancellationToken.None).GetAwaiter().GetResult())
```

Comment:
I've now set this to false because the tests which assert profile loading works do so by explicitly running the same code as would otherwise run in startup:

https://github.com/PowerShell/PowerShellEditorServices/blob/20a35163c39cceaee2023b49783bef6a0c4dfaec/src/PowerShellEditorServices/Services/PowerShell/Host/PsesInternalHost.cs#L217-L221

I don't think we _really_ need to be calling this delegate for every other test.